src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1304,6 +1423,8 @@ components:
       title: ConversationResponse
       type: object
       description: Represents a Conversation with a prisoner within the context of their Learning and Work Progress.
+      allOf:
```

Comment:
This is the existing type that I've changed to use the `AuditedResource` base type; we can do a similar thing for other types in the future.
I didnt want to do them all in this PR so as to try to keep this PR reasonably focussed on the new pre-prison education API endpoints 👍